src/injected/analyzer-controller.ts

```diff
@@ -82,27 +78,22 @@ export class AnalyzerController {
     };
 
     private initializeAnalyzers(): void {
-        EnumHelper.getNumericValues(VisualizationType).forEach((test: VisualizationType) => {
-            const config = this.visualizationConfigurationFactory.getConfiguration(test);
-            if (this.assessmentsProvider.isValidType(test)) {
-                this.assessmentsProvider.forType(test).requirements.forEach(stepConfig => {
-                    this.analyzers[stepConfig.key] = config.getAnalyzer(
-                        this.analyzerProvider,
-                        stepConfig.key,
-                    );
-                });
-            } else {
-                const key = config.getIdentifier();
-                this.analyzers[key] = config.getAnalyzer(this.analyzerProvider);
-            }
-        });
+        this.visualizationConfigurationFactory.forEachConfig(
+            (testConfig, type, requirementConfig) => {
```

Comment:
```suggestion
            (testConfig: VisualizationConfiguration, type: VisualizationType, requirementConfig: Requirement) => { => {
```

Explicit types increase readability.